shell/pages/home.vue

```diff
@@ -91,6 +91,10 @@ export default {
     ...mapGetters(['currentCluster', 'defaultClusterId', 'releaseNotesUrl']),
     mcm: mapFeature(MULTI_CLUSTER),
 
+    mgmtClusters() {
+      return this.$store.getters['management/all'](MANAGEMENT.CLUSTER);
```

Comment:
I'm curious how the unit test in `TopLevelMenu.test.ts` didn't catch this case? is it possible to create a test case to cover both scenarios(i.e. `pCluster?.description || x.description`)?

Reply:
Sure thing. Unit test added.